### PR TITLE
Test for CR

### PR DESCRIPTION
**Pull request by amiangshu**

### CODE REVIEW

**Comment by PattyCoding123** [comment]
I shall review these changes

**Review by PattyCoding123 — APPROVED** [review]
Good changes

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 [![MIT License][license-shield]][license-url]
 
 <!-- PROJECT LOGO -->
+
```
Comment (ExzyELI) [inline README.md:7]: Line not needed

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 [![MIT License][license-shield]][license-url]
 
 <!-- PROJECT LOGO -->
+
+It's just a placeholder.
```
Comment (amiangshu) [inline README.md:8]: Just a comment

**Review by ExzyELI — COMMENTED** [review]
Extra line not needed

File: README.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 [![MIT License][license-shield]][license-url]
```
Comment (amiangshu) [inline README.md:5]: More comment

**Pull request merged**